src/Pages/Onboarding/OnboardingMod4.jsx

```diff
@@ -28,63 +29,119 @@ const headerText = {
+    links: [
+        { linkTitle: "Pick a task Guide", linkIcon: "videoFindWork" },
+        {
+            linkTitle: "Misc. Tasks",
+            linkIcon: "link",
+            linkLocation: "https://airtable.com/appfJZShN8K4tcWHU/tblXQZfKPAJIjV4cL/viw5qPVwWi7GiDT2J?blocks=hide",
+        },
+    ],
 };
 
 const bugs = {
     title: "Bug reports",
     paragraph1:
         "Our website and tool are living creations requiring constant iteration and improvement. The ideas for this improvement come from bug reports created by you and the rest of the team! Here is the link you’ll use to create bug reports–please bookmark it!",
-    linkTitle: "Report bugs",
-    linkIcon: "link",
-    linkLocation: "https://airtable.com/shrzTUdyiDQsnqR3K",
-    linkTitle2: "Glossary",
-    linkIcon2: "link",
-    linkLocation2:
-        "https://airtable.com/appfJZShN8K4tcWHU/tblYLt2L646QsTESQ/viwAXRIfCUQNDNs7M/recYE5SnGWsEGsSf5?blocks=hide",
+    links: [{ linkTitle: "Report bugs", linkIcon: "link", linkLocation: "https://airtable.com/shrzTUdyiDQsnqR3K" }],
+};
+
+const ourTeams = {
+    title: "Our teams",
+    links: [
+        { linkTitle: "Airtable teams list", linkIcon: "link", linkLocation: "https://airtable.com/shrK9XGUYAHu8gy6k" },
+    ],
+};
+
+const teamDesc = {
+    title: "Team descriptions",
+    links: [
+        {
+            linkTitle: "Scrum",
+            linkIcon: "teamIcon",
+            linkImg: <img src={scrum} alt="" className="makeStyles-linkIcon-11" />,
+            modalTitle: "Scrum",
+            modalParagraph1:
+                "The Scrum team works in 2 week sprints where we focus on a few specific tasks each sprint.",
+            modalParagraph2:
+                "Interested? Join the “calculator scrum team” on slack. There you will see posts asking for volunteers for the next scrum team, and what disciplines we are looking for. Sign up by commenting and adding your name to this team!",
+            modalParagraph3: "Please note, only people who have finished their Agile learning task should join Scrum.",
+        },
+        {
+            linkTitle: "Marketing",
+            linkIcon: "teamIcon",
+            linkImg: <img src={marketing} alt="" className="makeStyles-linkIcon-11" />,
+            modalTitle: "Marketing",
+            modalParagraph1:
+                "We promote Clearviction’s mission, vision and causes to get the attention of potential volunteers, partners and donors. ",
+        },
+        {
+            linkTitle: "User Research",
+            linkIcon: "teamIcon",
+            linkImg: <img src={userResearch} alt="" className="makeStyles-linkIcon-11" />,
+            modalTitle: "User Research",
+            modalParagraph1:
+                "We provide relevant insights on how users interact with ClearViction, through UX methods including user interviews, usability testing, persona research, surveys and more.",
```

Comment:
What is the official "spelling" of clearviction? Is it `Clearviction` or `ClearViction`? Whichever it is, let's be consistent with it's usage throughout the site